bigchaindb/util.py

```diff
@@ -1,4 +1,12 @@
+
```

Comment:
yes, totally agree, I tried to refactor this part but was too much for this pull request.